### PR TITLE
Adapt to crust mainnet

### PR DESCRIPTION
**Pull request by mmyyrroonn**

### CODE REVIEW

File: chains/ethereum/events.go [hunk 1]
```diff
@@ -21,7 +23,7 @@ func (l *listener) handleErc20DepositedEvent(destId msg.ChainId, nonce msg.Nonce
 		l.cfg.id,
 		destId,
 		nonce,
-		record.Amount,
+		new(big.Int).Div(record.Amount, big.NewInt(1000000)),
```
Comment (mmyyrroonn) [inline chains/ethereum/events.go:26]: Here

File: chains/substrate/connection.go [hunk 2]
```diff
@@ -60,6 +60,9 @@ func (c *Connection) Connect() error {
 	if err != nil {
 		return err
 	}
+	opts := types.SerDeOptions{NoPalletIndices: true}
```
Comment (mmyyrroonn) [inline chains/substrate/connection.go:63]: Here

File: chains/substrate/events.go [hunk 3]
```diff
@@ -41,7 +41,7 @@ func fungibleTransferHandler(evtI interface{}, log log15.Logger) (msg.Message, e
 		0, // Unset
 		msg.ChainId(evt.Destination),
 		msg.Nonce(evt.DepositNonce),
-		evt.Amount.Int,
+		big.NewInt(0).Mul(evt.Amount.Int, big.NewInt(1000000)),
```
Comment (mmyyrroonn) [inline chains/substrate/events.go:44]: Here

File: cmd/chainbridge/account.go [hunk 4]
```diff
@@ -158,8 +159,14 @@ func importPrivKey(ctx *cli.Context, keytype, datadir, key string, password []by
 
 	if keytype == crypto.Sr25519Type {
 		// generate sr25519 keys
-		network := ctx.String(config.SubkeyNetworkFlag.Name)
-		kp, err = sr25519.NewKeypairFromSeed(key, network)
+		networkString := ctx.String(config.SubkeyNetworkFlag.Name)
```
Comment (mmyyrroonn) [inline cmd/chainbridge/account.go:162]: Here

**Review by badkk — APPROVED** [review]

**Pull request merged**